src/gui/widgets/multi_page.cpp

```diff
@@ -267,20 +283,25 @@ namespace implementation
 {
 
 builder_multi_page::builder_multi_page(const config& cfg)
-	: implementation::builder_styled_widget(cfg), builder(nullptr), data()
+	: implementation::builder_styled_widget(cfg), builders(), data()
 {
-	const config& page = cfg.child("page_definition");
+	const config& page = cfg.child();
 
-	VALIDATE(page, _("No page defined."));
-	builder = std::make_shared<builder_grid>(page);
-	assert(builder);
+	for (const config& page : cfg.child_range("page_definition"))
+	{
+		auto builder = std::make_shared<builder_grid>(page);
+		assert(builder);
+		builders[page["id"]] = builder;
+	}
+	VALIDATE(!builders.empty(), _("No page defined."));
 
 	/** @todo This part is untested. */
 	const config& d = cfg.child("page_data");
 	if(!d) {
 		return;
 	}
 
+	auto builder = builders.begin()->second;
```

Comment:
It looks like maybe this was never updated with I updated `list_data`. Not sure.